crates/consensus/src/transaction/mod.rs

```diff
@@ -59,6 +59,11 @@ pub trait Transaction: any::Any + Send + Sync + 'static {
 /// types, or in other networks. For example, in Optimism, the deposit transaction signature is the
 /// unit type `()`.
 pub trait SignableTransaction<Signature>: Transaction {
+    /// True if the transaction uses EIP-155 signatures.
+    fn use_eip155(&self) -> bool {
+        false
```

Comment:
should the default be true here?

Reply:
EIP155 signatures are disallowed for all types except legacy, no?

Reply:
right, misinterpreted the macro